bin/cli.js

```diff
@@ -2,12 +2,18 @@
 
 'use strict';
 
-var treeAsList = require('../');
+var dependencyTree = require('../');
 var filename = process.argv[2];
 var root = process.argv[3];
 
-var tree = treeAsList(filename, root);
+var tree = dependencyTree(filename, root);
 
-tree.forEach(function(node) {
+console.log('Pre-Order:');
```

Comment:
I often use the CLI's output in editor plugins like https://github.com/mrjoelkemp/sublime-dependents where the cleaner the output, the easier the integration.

If distinguishing between traversal methods was necessary, I'd opt for a cli flag `traversalMethod` with potential values: `preOrder` or `postOrder` to distinguish which method to call and avoid `console.log('\nPost-Order:');`.